R/roxygen2.R

```diff
@@ -0,0 +1,13 @@
+# THIS FILE INCLUDES FUNCTION THAT ARE RE-USED THROUGHOUT
+# roxygen2 COMMENTS TO GENERATE DOCUMENTATION TEXT
+
+roxygen_param_by_var <- function(arg) {
+  case_when(
+    arg == 1 ~ paste0("By Variables for Grouping: \n",
+                      "Used to categorize and aggregate data to produce a summary value \n",
+                      "to be inserted as a parameter or variable"),
+    arg == 2 ~ paste0("By Variables for Joining: \n",
```

Comment:
@pharmaverse/admiral For the most part, most of our functions `by_vars` statements parse into 2 buckets, commited two "generic text" prompts for now